userguide.md

```diff
@@ -164,6 +164,160 @@ export PATH=$PATH:/c/progra~1/PostgreSQL/8.2/bin
 USE_PGXS=1 make
 USE_PGXS=1 make install
 ```
+
+In Mingw, MSYS, or MSYS2:
+
+If R is built and installed using a sub-architecture, as explained in the section Sub-architectures in
+https://cran.r-project.org/doc/manuals/r-release/R-admin.html
+for example, in an R
+```
+R-x.y.z for Windows (32/64 bit)
+```
+that has been downloaded (and installed) from
+[https://cran.r-project.org/bin/windows/base/](https://cran.r-project.org/bin/windows/base/)
+
+then, include the environment variable R_ARCH.
+For example R_ARCH=/x64 (or R_ARCH=/i386 as appropriate):
+```
+export R_HOME=/c/progra~1/R/R-4.0.4
+export PATH=$PATH:/c/progra~1/PostgreSQL/13/bin
+export R_ARCH=/x64
+USE_PGXS=1 make
+USE_PGXS=1 make install
+```
+
+
+
+### Installing from a Pre-Built "plr"
+
+Win32 - adjust paths according to your own setup, and be sure to restart the PostgreSQL service after
+changing:
+
+In Windows environment (generally):
+```
+R_HOME=C:\Progra~1\R\R-3.5.1
```

Comment:
Same as above, R 4.10 (I realize I'm picking nits at this point)

Reply:
* yes, instead 4.10 can go there